HelpSource/Classes/Object.schelp

```diff
@@ -189,11 +189,15 @@ A String containing the resulting file's path.
 
 subsection::Equality and Identity
 
+Equality, queried with code::==:: or code::!=::, is whether the values or contents of two objects are the same by comparing the values, contents or emphasis::hash table indecise::. Identity, queried by code::===:: or code::!==::, is whether two objects refer to the same object by comparing the emphasis::identity hash table indecise:: or virtual memory addresses.
```

Comment:
[In the method .hash and .identityHash](https://doc.sccode.org/Classes/Object.html#-hash), there are the following explanations:
> Answer a code used to index into a hash table. 

Based on this, I researched "hash function" and "hash table". As far as I understand, "hash table index" means "the position or location of an element within a hash table data structure, determined by a hash function. It is an integer value representing the slot or bucket in which the element is stored". The following terms seem to be used instead of the term "hashed table index": 
hash value, hash code, hash key, hash address, hash slot, bucket index, hash table position, hash table slot, hash table address, hash table location

I thought "hash table index" would be the simplest term for readers who are not so familiar with this area, but I don't mind the terminology. The term "hash table index" can be replaced by any term that you or other administrators with expertise suggest.